src/Servers/IIS/IIS/src/Core/HttpRequestStream.cs

```diff
@@ -97,6 +70,18 @@ private async ValueTask<int> ReadAsyncInternal(Memory<byte> buffer, Cancellation
             }
         }
 
+        public override async Task CopyToAsync(Stream destination, int bufferSize, CancellationToken cancellationToken)
+        {
+            try
+            {
+                await _body.CopyToAsync(destination, cancellationToken);
```

Comment:
So everytime I tried to use CopyToAsync to copy one stream/pipe to another, I wasn't able to use CopyToAsync. It kept on hanging during the call to CopyToAsync. Do you know why that would be? Could there be an issue with CopyToAsync?

Reply:
Could be and it needs to be fixed if there's an issue (and since we're not at the end of a milestone anymore I figured it makes sense to use it)